modules/cert-manager/kustomization.yaml

```diff
@@ -2,8 +2,12 @@
 apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 
-namespace: cert-manager
+commonAnnotations:
+  argocd.argoproj.io/sync-wave: "1"
```

Comment:
What are the criteria we follow here? I believe we should put that as `2`

`1` => Controller components (Argo CD, Crossplane, and anything used in the bootstrap).
`2` => Core utliz like Cert-Manager, External-DNS, etc.